metering/metering-uninstall.adoc

```diff
@@ -1,13 +1,32 @@
 :context: metering-uninstall
 [id="metering-uninstall"]
-= Uninstalling Metering
+= Uninstalling metering
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 You can remove metering from your {product-title} cluster.
 
-include::modules/metering-uninstall.adoc[leveloffset=+1]
+[NOTE]
+====
+Metering does not manage or delete Amazon S3 bucket data. After uninstalling metering,
+you must manually clean up S3 buckets that were used to store metering data.
+====
+
+[id="metering-remove"]
+== Removing the Metering Operator from your cluster
 
+Remove the Metering Operator from your cluster by following the documentation on
+xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
 
+[NOTE]
+====
+Removing the Metering Operator from your cluster does not remove its custom resource
```

Comment:
Throughout most of this, you refer to them as `CustomResourceDefinitions`. I know we are a long ways from aligning on our preferred usage but here you might want to change to PascalCase for consistency in this section.